cmd/clair-load-test/reports.go

```diff
@@ -208,27 +214,35 @@ func orchestrateWorkload(ctx context.Context, manifests [][]byte, manifestHashes
 	if err != nil {
 		return fmt.Errorf("Error while running POST operation on index_report: %w", err)
 	}
+
+	zlog.Info(ctx).Msg("Sleeping for 5 minutes")
```

Comment:
Keeping only this timeout, because we observed index report retrying requests which is affecting the other API endpoint results which are dependent on index report POST operation. Adding this timeout will provide POST index_report endpoint with enough time to run requests and retry them before we move on to the subsequent endpoints that are dependent on this. Let us know if this doesn't make any sense. But we saw improvements in the tests results when we did this.